code/modules/research/designs/weapon_designs.dm

```diff
@@ -101,7 +101,7 @@
 	id = "temp_gun"
 	req_tech = list("combat" = 3, "materials" = 4, "powerstorage" = 3, "magnets" = 2)
 	build_type = PROTOLATHE
-	materials = list(MAT_METAL = 5000, MAT_GLASS = 500, MAT_SILVER = 3000)
+	materials = list(MAT_METAL = 5000, MAT_GLASS = 500)
```

Comment:
Giving the temp gun this massive of a QoL *and* dropping the silver cost is asking for trouble, IMO.

This means you can mass produce them in autolathes, too.